src/common/CopyrightUtils.js

```diff
@@ -0,0 +1,20 @@
+class CopyrightUtils {
+  static getMapCopyrights(map) {
+    const { sourceCaches } = map.style;
+    let copyrights = [];
+    Object.values(sourceCaches).forEach((sourceCache) => {
+      if (sourceCache.used) {
+        const source = sourceCache.getSource();
+        if (source.attribution) {
+          copyrights = copyrights.concat(
+            source.attribution.split(/(<a.*?<\/a>)/),
+          );
+        }
+      }
+    });
+
+    return [...new Set(copyrights.filter((c) => c.trim()))];
+  }
+}
+
+export default CopyrightUtils;
```

Comment:
Since it's mapbox related it should not be in common folder but in mapbox . I don't think a class is useful to declare utils function just export the function, it sim'lifies the code and make it more readable.

I like the way @friedjoff declare utils function : https://gitlab.geops.de/geops/kja-app/-/blob/master/utils/defineAbilityFor.js 

I find the one function/one file good.